packages/react/src/ActionList/ActionList.test.tsx

```diff
@@ -648,4 +648,52 @@ describe('ActionList', () => {
     await userEvent.keyboard('{ArrowUp}')
     expect(document.activeElement).toHaveTextContent('Option 4')
   })
+
+  describe('ActionList.Description', () => {
+    it('should render the description as inline without truncation by default', () => {
+      const {getByText} = HTMLRender(
+        <ActionList>
+          <ActionList.Item>
+            Item 1<ActionList.Description>Item 1 description</ActionList.Description>
+          </ActionList.Item>
+        </ActionList>,
+      )
+
+      const description = getByText('Item 1 description')
+      expect(description.tagName).toBe('SPAN')
+      expect(description).toHaveStyleRule('flex-basis', 'auto')
+      expect(description).not.toHaveStyleRule('overflow', 'ellipsis')
+      expect(description).not.toHaveStyleRule('white-space', 'nowrap')
+    })
+    it('should render the description as `Truncate` when truncate is true', () => {
+      const {getByText} = HTMLRender(
+        <ActionList>
+          <ActionList.Item>
+            Item 1<ActionList.Description truncate>Item 1 description</ActionList.Description>
+          </ActionList.Item>
+        </ActionList>,
+      )
+
+      const description = getByText('Item 1 description')
+      expect(description.tagName).toBe('DIV')
+      expect(description).toHaveAttribute('title', 'Item 1 description')
+      expect(description).toHaveStyleRule('flex-basis', '0')
+      expect(description).toHaveStyleRule('text-overflow', 'ellipsis')
+      expect(description).toHaveStyleRule('overflow', 'hidden')
+      expect(description).toHaveStyleRule('white-space', 'nowrap')
```

Comment:
Very detail tests!! Love it 🔥 

I’m also curious to learn when you prefer to use Jest for visual style testing versus a tool like Playwright. 👀

Reply:
That's a great question! I actually didn't take the fact that we already had a playwright test for the truncation story into account when writing these so this might be a bit "overkill" haha. I think I tend to go for Jest because it's what I'm most familiar with but love how "complete" vrt testing is in that it'll fail on the smallest visual change in the entire story so it feels like it covers more ground than a specific jest test. That being said I'm a big fan of redundancy.